test/tools/auth0/handlers/clientGrants.tests.js

```diff
@@ -34,7 +34,7 @@ describe('#clientGrants handler', () => {
       ];
 
       try {
-        await stageFn.apply(handler, [ { clientGrants: data } ]);
+        await stageFn.apply(handler, [{ clientGrants: data }]);
```

Comment:
Apologies with the formatting changes. This is something we'll have to standardize with linting.